rero_ils/config.py

```diff
@@ -4396,3 +4396,6 @@ def search_type(field):
 RERO_FILES_RECORD_FILE_SERVICE_CONFIG = (
     "rero_ils.modules.files.services.RecordFileServiceConfig"
 )
+RERO_ILS_APP_FILES_UI_MAX = 600
```

Comment:
You need to expose this here: https://github.com/rero/rero-ils/blob/977d293e3352ce3ce1f2a888543b88e36e7a3bde/rero_ils/modules/patrons/views.py#L131

Reply:
Added: `"maxFiles": config.get("RERO_ILS_APP_FILES_UI_MAX", 500),`